packages/core/src/Chip/ButtonChip/ButtonChip.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space, fontSize } from 'styled-system'
+import { ChevronDown } from 'pcln-icons'
+import { ChipContent } from '../ChipContent'
+import { ChipContentWrapper } from '../ChipContentWrapper'
+import { Button } from '../../Button'
+import { getPaletteColor } from '../../utils'
+
+const ChipButton = styled(Button)`
+  border: none;
+  padding: 0;
+  &:focus > ${ChipContentWrapper} {
+    box-shadow: 0 0 0 1px ${getPaletteColor('base')};
+    border-color: ${getPaletteColor('base')};
+    background-color: ${getPaletteColor('light')};
+  }
+`
+
+const getTitle = ({ disabled, expanded }) =>
+  disabled ? 'Disabled' : expanded ? 'Collapse' : 'Expand'
+
+const ButtonChip = ({
+  id,
+  m,
+  disabled,
+  expanded,
+  children,
+  onClick,
+  label,
+  showActionIcon,
+  ...props
+}) => (
+  <ChipButton
+    id={id}
+    disabled={disabled}
+    onChange={onClick}
+    aria-expandaded={expanded}
+    m={m}
+  >
+    <ChipContent
+      label={label}
+      disabled={disabled}
+      selected={expanded}
+      action={showActionIcon && { Icon: ChevronDown, title: getTitle(props) }}
+      {...props}
```

Comment:
Since you‘re passing `m` as a prop to ChipButton, it will also get applied to ChipContent here. Is that intentional?